app/templates/components/sales-table.hbs

```diff
@@ -0,0 +1,55 @@
+<div class="ui stackable grid">
```

Comment:
Move this component into `admin/sales/sales-table`

Reply:
How will the table render than, As for Events Table too, We have defined `UI Table` in components only, not explicitly into the folder

Reply:
What do you mean ? events-table is defined inside an `events` folder ..

Reply:
@niranjan94 Moving the `components/sales-table.js` into the `components/admin/sales` folder is not rendering the values and showing empty table

Reply:
Then find out what you're doing wrong ... Just saying _not working_ isn't gonna make it work. Nesting components in subfolders is a very basic functionality that is a core of ember and is already used. So, the only reason it may not work is if you're doing it wrong. 

If you're still not able to figure it out, do push the code to the branch. Will review and help out,

Reply:
@niranjan94 did the nested templates, And now the data in templates is not getting rendered, And I think its due to because all the tables data I have defined in `Admin/Controllers` or something else it is.
Please help

Reply:
@niranjan94 did the nested templates, And now the data in templates is not getting rendered, And I think its due to because all the tables data I have defined in Admin/Controllers or something else it is.
Please help

Reply:
@kushthedude a component = component js + component template.
Moving a component into a sub-directory would mean moving _both_ the js file and the template file.

Reply:
@kushthedude I suggest you learn more about the basics of ember.js before working on any more issues.

Reply:
In case you don't understand something, feel free to ping me on gitter. Will help.

Reply:
@niranjan94 I did that but that completely broke the page, After seeing logs it said `Sales-Table` is not a valid helper, and to make the table work again : I think i need to move the `Admin/sales`  templates files into the new folder in components I created, Please tell me if i am wrong 
 
> @kushthedude
> 
> `a component = component js + component template`
> 
> Moving a component into a sub-directory would mean moving _both_ the js file and the template file.

Reply:
If component is at `admin/sales/sales-table` ... then you should also refer to it by `{{admin/sales/sales-table}}` instead of `{{sales-table}}`